scripts/bootloader/provision.py

```diff
@@ -63,7 +68,10 @@ def get_hashes(public_key_files):
     hashes = list()
     for fn in public_key_files:
         with open(fn, 'rb') as f:
-            hashes.append(sha256(VerifyingKey.from_pem(f.read()).to_string()).digest()[:16])
+            key = sha256(VerifyingKey.from_pem(f.read()).to_string()).digest()[:16]
+            if key_contains_ffff(key):
+                raise RuntimeError("Public key containing 0xffff found. Please try again with a valid key")
```

Comment:
Ideally we should tell the user a) which key triggered this, and b) that it's the hash that has ffff, not the key itself.